expand/expand_metadata.go

```diff
@@ -23,7 +23,7 @@ func ExpandMetadata(in []interface{}) metav1.ObjectMeta {
 	if meta.Annotations == nil {
 		meta.Annotations = make(map[string]string)
 	}
-	meta.Annotations["confirmation.garden.sapcloud.io/deletion"] = "true"
+	meta.Annotations["confirmation.core.gardener.cloud/deletion"] = "true"
```

Comment:
Guys, the annotations are still not migrated the the new API group. See https://github.com/gardener/gardener/issues/1649.
Please revert the similar changes.

Reply:
thanks! we also found this issue in https://github.com/kyma-incubator/terraform-provider-gardener/pull/39#issuecomment-575158116, will prepare to revert the changes

Reply:
Hi @neo-liang-sap, @ialidzhikov,
it seems this change is causing some issues on the provider and cluster provisioning started failing in some of our jobs.

I asked on the gardener team channel and they gave me this resource:
https://listserv.sap.corp/pipermail/kubernetes-users/2019-October/000692.html

Please revert this PR and send a new one with the whole API version migration when ready.

Thanks!

Reply:
Hi @clebs :

Thanks!

The revert PR is https://github.com/kyma-incubator/terraform-provider-gardener/pull/42

Reply:
@clebs , my comment was to the annotations. What's the problem to use the new API group and but the annotations to be related the old API group?
We are looking forward to deprecate the old `garden.sapcloud.io` API group. Please kindly adapt your machinery to `core.gardener.cloud` API group.
Please refer to https://github.com/gardener/gardener/blob/v0.34.0/example/90-shoot.yaml for an example Shoot manifest.